<launch>

<node name="rviz" pkg="rviz" type="rviz" args="-d $(find robot_ui)/config.rviz" output="screen"/>

<node name="NMotionCtrlTopic" pkg="N_Robot_Topic" type="NMotionCtrl_X64_Topic" args="/dev/ttyUSB0"/>

<node name="robot_base_odometry" pkg="robot_base_odometry" type="robot_base_odometry_node" output="screen"/>
<node name="iqr" pkg="pan_tilt_driver" type="PanTiltDriverNode" output="screen">
    <param name="port_name" value="/dev/ttyACM0"/>
    <param name="yaw_joint_name" value="pan_tilt_yaw_joint"/>
    <param name="pitch_joint_name" value="pan_tilt_pitch_joint"/>
</node>

</launch>
